src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -193,6 +193,7 @@ namespace AppInstaller::CLI
             };
 
             info << std::endl << Resource::String::Logs << ": "_liv << Runtime::GetPathTo(Runtime::PathName::DefaultLogLocationForDisplay).u8string() << std::endl;
+            info << std::endl << Resource::String::UserSettings << ": "_liv << Runtime::GetPathTo(Runtime::PathName::UserSettingsFileLocation).u8string() << std::endl;

```

Comment:
>info << std::endl << Resource::String::UserSettings << ": "_liv << Runtime::GetPathTo(Runtime::PathName::UserSettingsFileLocation).u8string() << std::endl; [](http://example.com/codeflow?start=12&length=155)

We may need to remember to update this one when [https://github.com/microsoft/winget-cli/pull/2454](https://github.com/microsoft/winget-cli/pull/2454) is merged